common/src/main/webapp/sass/explore-common.scss

```diff
@@ -1,5 +1,5 @@
 $tile-title-height: 36px;
-$tree-section-width: 300px;
+$tree-section-width: 320px;
```

Comment:
Could yoeu verify this number is not used inother places in the code?

Reply:
Seems to be safe.